YDS-Storybook/SwiftUI/Atom/BadgePageView.swift

```diff
@@ -0,0 +1,64 @@
+//
+//  BadgePageView.swift
+//  YDS-Storybook
+//
+//  Created by 성현주 on 10/31/23.
+//
+
+import SwiftUI
+import YDS_SwiftUI
+
+
+
+struct BadgePageView: View {
```

Comment:
사소한 부분이긴 한데, Lint가 Whitespace Violation 경고를 띄우네요!

```suggestion
import YDS_SwiftUI

struct BadgePageView: View {
```